scripts/check-qemu-install

```diff
@@ -283,7 +312,8 @@ if not os.path.isfile(args.iso):
 
 if not os.path.exists('/dev/kvm'):
     log.error('KVM not enabled on host, proceeding with software emulation')
-    sys.exit(1)
+    # Some arm platforms do not have kvm, for example Mac arm devices
```

Comment:
This will change the behavior also for regular amd64 tests which is not a good idea. If MAC behaves different - add a condition to this check.


```
>>> import sys
>>> sys.platform
'linux'
```

Reply:
Another example - git arm64 runners do not support KVM